playbooks/templates/.env.j2

```diff
@@ -131,6 +131,14 @@ SKYNET_DASHBOARD_URL=https://account.{{ cluster_domain }}
 # Accounts log level
 SKYNET_ACCOUNTS_LOG_LEVEL=info
 
+{% if webportal_common_config.accounts_limit_access is defined %}
+# Accounts limit access to authenticated users
+# Possible values:
+# - 'authenticated': Only authenticated users can access the portal (default if this env variable is missing)
+# - 'false' or empty: All users incl. anonymous users can access the portal (access to portal is public)
```

Comment:
We have this in docker-compose:
```
- ACCOUNTS_LIMIT_ACCESS=${ACCOUNTS_LIMIT_ACCESS:-authenticated} #
```
which I think means that the empty value will be replaced with `authenticated`. I'm not 100% sure about that.

Reply:
Oh, you are right, I have read a comment on the variable, not the code.